tendermint/spn
# feat(reward): Implement `MsgSetRewards` message

Closes #477

### What does this PR does?

Add the `MsgSetReward` implementation to allow a coordinator to set a reward pool for a chain.

### How to test?

- Create the coordinator and chain
```shell
spnd tx profile create-coordinator --from alice && \
spnd tx launch create-chain orbit-1 orbit.com 0xaaa --from alice
```

- Create the reward
```shell
spnd tx reward set-rewards 1 1000uspn 100000 --from alice
```

- Try to create a reward token with a no-balance account. It should return an error
```shell
spnd tx reward set-rewards 1 1000token 100000 --from alice
```
